src/overlays/actors/ovl_En_Invadepoh_Demo/z_en_invadepoh_demo.c

```diff
@@ -15,14 +27,72 @@ void EnInvadepohDemo_Destroy(Actor* thisx, PlayState* play);
 void EnInvadepohDemo_Update(Actor* thisx, PlayState* play);
 void EnInvadepohDemo_Draw(Actor* thisx, PlayState* play);
 
-void func_80C19AB4(EnInvadepohDemo* this, PlayState* play);
-void func_80C19D00(EnInvadepohDemo* this, PlayState* play);
-void func_80C19D48(EnInvadepohDemo* this, PlayState* play);
-void func_80C19E04(EnInvadepohDemo* this, PlayState* play);
-void func_80C19EC0(EnInvadepohDemo* this, PlayState* play);
-void func_80C19F7C(EnInvadepohDemo* this, PlayState* play);
+void EnInvadepohDemo_DoNothing(EnInvadepohDemo* this, PlayState* play);
+void EnInvadepohDemo_Alien_Init(EnInvadepohDemo* this, PlayState* play);
+void EnInvadepohDemo_Romani_Init(EnInvadepohDemo* this, PlayState* play);
+void EnInvadepohDemo_Cow_Init(EnInvadepohDemo* this, PlayState* play);
+void EnInvadepohDemo_Ufo_Init(EnInvadepohDemo* this, PlayState* play);
+void EnInvadepohDemo_CowTail_Init(EnInvadepohDemo* this, PlayState* play);
+void EnInvadepohDemo_Cow_Destroy(EnInvadepohDemo* this, PlayState* play);
+void EnInvadepohDemo_CowTail_Destroy(EnInvadepohDemo* this, PlayState* play);
+void EnInvadepohDemo_Alien_Idle(EnInvadepohDemo* this, PlayState* play);
+void EnInvadepohDemo_Alien_FollowPath(EnInvadepohDemo* this, PlayState* play);
+void EnInvadepohDemo_Romani_Idle(EnInvadepohDemo* this, PlayState* play);
+void EnInvadepohDemo_Romani_FollowPath(EnInvadepohDemo* this, PlayState* play);
+void EnInvadepohDemo_Cow_Idle(EnInvadepohDemo* this, PlayState* play);
+void EnInvadepohDemo_Cow_FollowPath(EnInvadepohDemo* this, PlayState* play);
+void EnInvadepohDemo_Ufo_Idle(EnInvadepohDemo* this, PlayState* play);
+void EnInvadepohDemo_Ufo_FollowPath(EnInvadepohDemo* this, PlayState* play);
+void EnInvadepohDemo_SelectCueAction(EnInvadepohDemo* this, PlayState* play);
+void EnInvadepohDemo_CowTail_Idle(EnInvadepohDemo* this, PlayState* play);
+void EnInvadepohDemo_Alien_WaitForObject(EnInvadepohDemo* this, PlayState* play);
+void EnInvadepohDemo_Romani_WaitForObject(EnInvadepohDemo* this, PlayState* play);
+void EnInvadepohDemo_Cow_WaitForObject(EnInvadepohDemo* this, PlayState* play);
+void EnInvadepohDemo_CowTail_WaitForObject(EnInvadepohDemo* this, PlayState* play);
+void EnInvadepohDemo_Alien_Draw(EnInvadepohDemo* this, PlayState* play);
+void EnInvadepohDemo_Romani_Draw(EnInvadepohDemo* this, PlayState* play);
+void EnInvadepohDemo_Cow_Draw(EnInvadepohDemo* this, PlayState* play);
+void EnInvadepohDemo_Ufo_Draw(EnInvadepohDemo* this, PlayState* play);
+void EnInvadepohDemo_CowTail_Draw(EnInvadepohDemo* this, PlayState* play);
+
+#define DRAW_FLAG_SHOULD_DRAW 1
+#define EN_INVADEPOH_DEMO_CUE_ID_UNINITIALIZED -1
```

Comment:
```suggestion
#define EN_INVADEPOH_DEMO_CUE_ID_NONE -1
```
I think this is more common (also the benefit of being shorter)